README.md

```diff
@@ -8,6 +8,12 @@ This client aims for:
 - Advanced customizability (settings and userscripts)
 - Constructive to the community (open source and MIT license)
 
+this forks extra features:
+- electron version 9.4.4
+- yeeted menutimer
+  
+should be up to date with mixaz's master
+
```

Comment:
These changes are pretty much unnecessary (or at least needs to be polished)

Reply:
sorry my bad i wasnt aware these go to this pull request, i kinda just made my own fork lol